modules/data.atmosphere/R/met.process.R

```diff
@@ -153,7 +149,7 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
       cf.id <- convert.input(cf1.id, outfolder2,formatname,mimetype,site.id=site$id,start_date,end_date,pkg,permute.nc,
                              username,con=con,hostname=host$name,browndog=NULL,write=TRUE)
     }
-    cf.id <- 1000000023 #ID of permuted CF files in bety - they already exist on pecan2 and geo
+    cf.id <- list(input.id=1000000023, dbfile.id=NA) #ID of permuted CF files in bety - they already exist on pecan2 and geo
```

Comment:
This ties into my response to the previous comment. I believe this part will get cleaned up too.